app/code/VConnect/Blog/Api/PostRepositoryInterface.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace VConnect\Blog\Api;
+
+use VConnect\Blog\Api\Data\PostInterface;
+use Magento\Framework\Exception\CouldNotSaveException;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Framework\Exception\CouldNotDeleteException;
+
+interface PostRepositoryInterface
+{
+    /**
+     * @param PostInterface $post
+     * @return PostInterface
+     * @throws CouldNotSaveException
+     */
+    public function save(PostInterface $post);
+
+    /**
+     * @param $value
+     * @param string $field
+     * @return PostInterface
+     * @throws NoSuchEntityException
+     */
+    public function get($value, string $field = PostInterface::ENTITY_ID);
+
+//    /**
+//     * @param \Magento\Framework\Api\SearchCriteriaInterface $searchCriteria
+//     * @return \Magento\Cms\Api\Data\PageSearchResultsInterface
+//     * @throws \Magento\Framework\Exception\LocalizedException
+//     */
+//    public function getList(\Magento\Framework\Api\SearchCriteriaInterface $searchCriteria);
```

Comment:
what is the purpose of this commented phpdoc?

Reply:
In one audio I told you that when i studied question about model,resource model and collection, and our electricity dropped, my brother decided to help me better understand. And during that time when he showed me something and explained, he told to comment this part(something like 'You dont need it now'). 
When i started to create model,resource model and collection in VConnect_Blog model, I decided to keep it in order that you maybe tell me to use this method or to delete this commented code part.